static/templates/gear_menu.hbs

```diff
@@ -114,7 +114,7 @@
             {{/if}}
             {{#if show_plans}}
             <li role="presentation" class="hidden-for-spectators">
-                <a href="/plans" target="_blank" rel="noopener noreferrer" role="menuitem">
+                <a href="https://zulip.com/plans" target="_blank" rel="noopener noreferrer" role="menuitem">
```

Comment:
This one should remain `href="/plans"` so the user sees the selected plan and upgrade link for their realm. We only turn on `show_plans` if `CORPORATE_ENABLED`, so it’s safe to leave this relative.